modules/contributing.adoc

```diff
@@ -0,0 +1,46 @@
+[id="contributing-to-docs-contributing"]
+= Contribute to Hybrid Cloud Patterns documentation
+:icons:
+:description: Contribute to Hybrid Cloud Patterns documentation
+:imagesdir: ../images
+
+
+== Different ways to contribute
+
+There are a few different ways you can contribute to Hybrid Cloud Patterns documentation:
+
+* Email the Hybrid Cloud Patterns team at `<>@redhat.com`.
+* Create a link:https://github.com/hybrid-cloud-patterns/docs/issues[GitHub] or link:https://issues.redhat.com/projects/MBP/issues[Jira issue] .
+* Submit a pull request (PR). To create a PR, create a local clone of your own fork of the link:https://github.com/hybrid-cloud-patterns/docs[Hybrid Cloud Patterns docs repository], make your changes, and submit a PR. This option is best if you have substantial changes. For more details on creating a PR see <topic_link_to_contribution_workflow>.
+
+== Contribution workflow
+
+When you submit a PR, the [Hybrid Cloud Patterns Docs team] reviews the PR and arranges further reviews by Quality Engineering (QE), subject matter experts (SMEs), and others, as required. If the PR requires changes, updates, or corrections, the reviewers add comments in the PR. When the PR has been reviewed, all updates are complete, and all commits are squashed, the documentation team merges the PR.
```

Comment:
> [Hybrid Cloud Patterns Docs team]

What's the intention with the square brackets?

> When the PR has been reviewed, all updates are complete, and all commits are squashed, the documentation team merges the PR.

This sentence reads a little awkwardly due to the comma-separated list of subordinate clauses followed by the comma and main clause. Might also be a good idea to make the subordinate clauses  in active voice.

```suggestion
When you submit a PR, the [Hybrid Cloud Patterns Docs team] reviews the PR and arranges further reviews by Quality Engineering (QE), subject matter experts (SMEs), and others, as required. If the PR requires changes, updates, or corrections, the reviewers add comments in the PR. The documentation team merges the PR after they have reviewed the PR, you have implemented all feedback, and you have squashed all commits.
```

Reply:
> > [Hybrid Cloud Patterns Docs team]
> 
> What's the intention with the square brackets?
> 
This is again a placeholder. I plan to create a [team](https://github.com/orgs/hybrid-cloud-patterns/teams) for docs so that it's easier for anyone to tag us or notify us when GH issues are reported. I've added a comment in the relevant section.

Reply:
UPDATE: While creating a team, I noticed that a `docs` team already exists. So, I have now used that link and updated the file